store/cassandra/db.go

```diff
@@ -193,11 +193,10 @@ func (s *CassandraClient) Push(key model.Key, item store.OwnableItem) error {
 func (s *CassandraClient) Get(key model.Key) (store.OwnableItem, error) {
 	item, err := s.client.Get(key)
 	if err != nil {
-		if err == noDataResponse {
-			return item, store.KeyNotFoundError{Key: key}
+		if !errors.Is(err, store.ErrItemNotFound) {
```

Comment:
This change means there is no inclusion of `ItemNotFound` results in the metric - only success and all other failures.  Is that what we want?